packages/form-client/package.json

```diff
@@ -33,6 +33,7 @@
     "@crossbreeze/core": "0.0.0",
     "@crossbreeze/model-service": "^1.0.0",
     "@crossbreeze/protocol": "0.0.0",
+    "@crossbreeze/react-model-ui": "0.0.0",
     "@emotion/react": "^11.10.6",
     "@emotion/styled": "^11.10.6",
     "@mui/material": "^5.13.5",
```

Comment:
Yes, we can, good catch! Also in the property-view.

Reply:
Ok, I will pick this up later

Reply:
@martin-fleck-at  I am trying to set the right dependencies in the package.json files, but I am strugling to understand the react dependency.

Throughout the project I see 2 different type of React imports:

1. import * as React from '@theia/core/shared/react';
2. import * as React from 'react';

If the first one is used there is a dependency on Theia for this package (which is ok, as long as Theia is used on the packahe context). In the second scenario we use 'react', but it's never listed as a dependency in any package.json file.

Shound'nt we have one or the other everywhere? And whats the preferred option?

Reply:
Good catch! So we definitely should add `react` as a dependency to our `package.json` of the react-model-ui package but ideally we align with the version that Theia is providing as well (18.2.0 -- `yarn why react`). Theia is [re-exporting](https://github.com/eclipse-theia/theia/tree/master/packages/core#re-exports-mechanism) some common dependencies to make them available through their `@theia/core` package.

However, we purposefully avoided any Theia dependency in the `react-model-ui` package so that - in theory - it could even be re-used outside of Theia. And there really isn't anything Theia-specific that we need either, so in that package using `import * as React from 'react'` is the better approach, given that we actually define that dependency in the package.json.